frontend/packages/kubevirt-plugin/src/components/boot-order/boot-order-summary-empty-state.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { ListItem, Expandable, Text, TextVariants } from '@patternfly/react-core';
+import { BootableDeviceType } from '../../types';
+import { deviceLabel, deviceKey, bootOrderEmptyTitle, bootOrderEmptyMessage } from './constants';
+
+import './boot-order-summary.scss';
+
+export const BootOrderSummaryEmptyState = ({ devices }: BootOrderSummaryEmptyStateProps) => {
```

Comment:
I think we are trying to use `BootOrderSummaryEmptyState: React.FC<BootOrderSummaryEmptyStateProps>`  convention in all of our code